frontend/src/app/libs/common-components/create-draft-event-popover/create-draft-event-popover.component.html

```diff
@@ -0,0 +1,27 @@
+<div class="event-popover">
+    <form mat-dialog-content  [formGroup]="eventPopoverForm">
+        <mat-form-field class="event-popover__title-field">
+            <mat-label>Title</mat-label>
+            <input matInput
+                type="text"
+                placeholder="My title" 
+                formControlName="title">
+            <mat-error>
+                Please enter some title
+            </mat-error>
+        </mat-form-field>
+
+    </form>
+    
+    <div mat-dialog-action class="event-popover__btn-wrap">
+        <button 
+            mat-button 
+            class="event-popover__btn"
+            type="submit"
+            (click)=submitEventPopover()
+            [disabled]="eventPopoverForm.controls['title'].invalid"
+            [class.disabed]="eventPopoverForm.controls['title'].invalid">
```

Comment:
typo `.disabled`

Reply:
also you adding this class, but a not seeing it in scss file, do we really need it?